cli/cmd/simulation/quicksim.hpp

```diff
@@ -151,9 +176,13 @@ class quicksim_command : public command
      */
     fiction::quicksim_params params{};
     /**
-     * Simulation result.
+     * Simulation result for H-Si(100)-2x1 surface.
+     */
+    fiction::sidb_simulation_result<fiction::sidb_100_cell_clk_lyt> sim_result{};
+    /**
+     * Simulation result for H-Si(111)-1x1 surface.
      */
-    fiction::sidb_simulation_result<fiction::sidb_cell_clk_lyt> sim_result{};
+    fiction::sidb_simulation_result<fiction::sidb_111_cell_clk_lyt> sim_result_111{};
```

Comment:
Due to an `alice` bug, you might need to reset both of these variables to empty states every time before the function returns.

Reply:
Otherwise, they get stuck with their respective values between calls